cfme/fixtures/cli.py

```diff
@@ -17,13 +22,40 @@
 from cfme.utils.conf import cfme_data
 from cfme.utils.conf import credentials
 from cfme.utils.log import logger
+from cfme.utils.log_validator import LogValidator
 from cfme.utils.providers import list_providers_by_class
+from cfme.utils.single import single
 from cfme.utils.version import Version
 from cfme.utils.wait import wait_for
 
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+try:
+    from urlparse import urljoin
+except ImportError:
+    from urllib.parse import urljoin
+
+
 TimedCommand = namedtuple("TimedCommand", ["command", "timeout"])
 
 
+@contextmanager
+def waiting_for_ha_monitor_started(appl, standby_server_ip, timeout):
+    if appl.version < '5.10':
```

Comment:
it's not needed anymore...

Reply:
@lina-nikiforova I think it is necessary here, because the test will upgrade from 5.9 to 5.10. Is that correct @jaryn?

Reply:
oh, @john-dupuy I think you are right